### PR TITLE
Repeating UD values

### PR DESCRIPTION
**Pull request by g2hchang**
This is related to #47 
I have implemented users to define number of repeats and absolute length value.
Initially, the repeat counts = 2 and absolute length = 0 as a default value : 
screen shot 2017-11-05 at 11 18 35 pm

Then user can set the value to the value they wish to define as such  : 
screen shot 2017-11-05 at 11 19 23 pm

### CODE REVIEW

**Comment by amgrubb** [comment]
Sorry for the delay, are the boxes "text" boxes or are they "up/down" integer boxes?
If "up/down" integer boxes, did you figure out how to show "N/A" in Absolute Length?

**Comment by g2hchang** [comment]
Both boxes are input type="number", so they have "up/down" integer boxes. Also, only way to have "N/A" is to have input type="text". I have added the note to indicate what 0 means in absolute length in red text.

**Comment by amgrubb** [comment]
This is great, I want to add some stuff to the branch before merging.

**Comment by amgrubb** [comment]
@g2hchang  I pushed a few changes. There is still two things I need you to do.
1. InputDynamics.js line 75-76 need to be finished.
2. ElementInspector.js line 143 can you add a bit of whitespace so that the note can naturally break the line after the word Length. "Note: Leave \"Absolute Length\" as 0 for unknown length."

File: leaf-ui/js/object/InputDynamic.js [hunk 1]
```diff
@@ -72,6 +72,10 @@ function getDynamics(){
 				}else{
 					line += "\tR\t" + rBegin + "\t" + rEnd;
 				}
+				// TODO Add the repeat count value and the absolute length value here
+				console.log(typeof $("#repeat-end2").val());
+				line += "\t" + $("#repeat-end2").val() + "\t" + $("#repeat-end3").val();
+				console.log(line);
```
Comment (amgrubb) [inline leaf-ui/js/object/InputDynamic.js]: Do you want all these console.log message left in?

**Pull request merged**